src/hooks/screen/useChatScreenHook.ts

```diff
@@ -8,11 +8,14 @@ import {v4 as uuid} from 'uuid';
 
 import AnonymousMessageRepo from '../../service/repo/anonymousMessageRepo';
 import ChatSchema from '../../database/schema/ChatSchema';
+import ImageUtils from '../../utils/image';
 import SignedMessageRepo from '../../service/repo/signedMessageRepo';
 import UseChatScreenHook from '../../../types/hooks/screens/useChatScreenHook.types';
+import UserSchema from '../../database/schema/UserSchema';
 import useChatUtilsHook from '../core/chat/useChatUtilsHook';
 import useLocalDatabaseHook from '../../database/hooks/useLocalDatabaseHook';
 import useMessageHook from './useMessageHook';
+import useUserAuthHook from '../core/auth/useUserAuthHook';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import `ImageUtils` has been removed in the new hunk. If this utility is not used anywhere else in the file, then this change is fine. However, if it's used elsewhere, this could cause a reference error.
```diff
- import ImageUtils from '../../utils/image';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->